### PR TITLE
Agent: use OpenShift version extracted from the release payload

### PR DESCRIPTION
**Pull request by andfasano**
Configure the `ClusterImageSet` manifest using the version fetched from the downloaded release payload

### CODE REVIEW

**Comment by zaneb** [comment]
I don't think anything looks at the name (even in principle they only have to match), but this seems fine.

**Comment by zaneb** [comment]
/hold
This will break my hacky workaround to fix CI in openshift/installer#5954, so we shouldn't merge it until CI is fixed properly in https://github.com/openshift/installer/pull/5949.

**Comment by zaneb** [comment]
/hold cancel
I tested and it appears nothing looks at the name at all as of now.

File: agent/04_agent_configure.sh [hunk 1]
```diff
@@ -64,6 +64,9 @@ function generate_cluster_manifests() {
   # To be change in ${OCP_DIR}/cluster-manifests
   MANIFESTS_PATH="${OCP_DIR}/manifests"
 
+  # Fetch current OpenShift version from the release payload
+  VERSION="$(openshift_version ${OCP_DIR})"
```
Comment (rwsu) [inline agent/04_agent_configure.sh:68]: I'm seeing an error in my local test of "make agent":
```
++(./agent/04_agent_configure.sh:68): generate_cluster_manifests(): openshift_version ocp/ostest
./agent/04_agent_configure.sh: line 68: openshift_version: command not found

```
I think this needs to be added to the beginning of the file to include the openshift_version function:
`source $SCRIPTDIR/ocp_install_env.sh`

**Review by rwsu — CHANGES_REQUESTED** [review]

File: agent/04_agent_configure.sh (hunk 1 above)
Comment (andfasano) [reply agent/04_agent_configure.sh:68]: Thanks @rwsu

**Pull request merged**